include/CommonAPI/DBus/DBusMessage.hpp

```diff
@@ -94,6 +94,7 @@ class COMMONAPI_EXPORT DBusMessage {
     bool setDestination(const char* destination);
 
     void setSerial(const unsigned int serial) const;
+    void setNoReplyExpected(bool replyNotExpected);
```

Comment:
How about setReplyExpected instead? (See later comment).

Unless you are sure it will never be called to set reply expected, always to set reply not expected.  In that case the parameter can be dropped.